scipy/stats/_continuous_distns.py

```diff
@@ -6858,9 +6858,18 @@ def _pdf(self, x, a):
     def _cdf(self, x, a):
         return x*(a + x**a)**(-1.0/a)
 
+    def _sf(self, x, a):
+        lg = sc.xlog1py(-1.0 / a, a * x**-a)
+        return -sc.expm1(lg)
+
     def _ppf(self, q, a):
         return (a/(q**-a - 1.0))**(1.0/a)
 
+    def _isf(self, q, a):
```

Comment:
This, too, has trouble with large `a`.  Try moderate values of `q` with large (e.g. `1e5`) values of `a`.